app/javascript/mastodon/components/column_header.tsx

```diff
@@ -9,7 +9,8 @@ import ArrowBackIcon from '@/material-icons/400-24px/arrow_back.svg?react';
 import ChevronLeftIcon from '@/material-icons/400-24px/chevron_left.svg?react';
 import ChevronRightIcon from '@/material-icons/400-24px/chevron_right.svg?react';
 import CloseIcon from '@/material-icons/400-24px/close.svg?react';
-import SettingsIcon from '@/material-icons/400-24px/settings.svg?react';
```

Comment:
`NotificationsPermissionBanner` still uses the old icon

Reply:
Thanks for reporting, just opened a PR here: https://github.com/mastodon/mastodon/pull/35193